mcpserver/args.go

```diff
@@ -0,0 +1,40 @@
+package mcpserver
+
+import "github.com/mark3labs/mcp-go/mcp"
+
+var (
+	explainationArgument = mcp.WithString("explanation",
+		mcp.Description("One sentence explanation for why this directory is being listed."),
```

Comment:
🎨 **Style (Low):** Generic description text 'why this directory is being listed' doesn't match all use cases. This description is used for environment creation, file operations, etc.

```suggestion
		mcp.Description("One sentence explanation for why this operation is being performed."),
```